.github/workflows/release.yml

```diff
@@ -573,10 +573,48 @@ jobs:
     needs: [vars, prepare]
     steps:
       - name: Check status
+        id: check-status
         env:
           RESULTS: "${{ toJSON(needs.*.result) }}"
         run: "[[ $(jq -n 'env.RESULTS | fromjson | unique == [\"success\"]') == \"true\" ]]\n"
 
+      - name: Post mattermost message
+        if: always()
```

Comment:
What is the purpose of this line?

Reply:
from here: https://docs.github.com/en/actions/learn-github-actions/expressions#always

> Causes the step to always execute, and returns true, even when canceled. A job or step will not run when a critical failure prevents the task from running. For example, if getting sources failed.